### ISSUE TITLE
Replace keccak_bit.rs with keccak_packed_multi.rs

### ISSUE THREAD

**Issue by ChihChengLiang** [privacy-scaling-explorations/zkevm-circuits#809]
This is an issue brought up by @lispc 

The TLDR is keccak_packed_multi works better with the current aggregation circuit.

The keccak_bit use ~1700 columns. It was based on the assumption that the column or curve point is only multiplied with a 1-bit scalar so that we have cheap multi-scalar multiplication(MSM) operations. It is assumed the MSM is performed on bare metal.

But the cheap MSM was not the case for running in the aggregation circuit. The aggregation circuit assumes the scalar is 254 bits and it verifies the complete 254-bit scalar multiplication anyway.

We could customize a 1-bit scalar multiplication aggregation circuit, but that's another work. Swapping to use the keccak_packed_multi.rs, which has only ~200 columns, is the simpler way.

See https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/634#issuecomment-1212143524 for performance comparison.

cc @han0110 @Brechtpd

**Comment by ChihChengLiang** [comment]
@Brechtpd or is there a way to configure keccak_bit to trade columns into rows?

**Comment by Brechtpd** [comment]
Currently no option to do so no, but it's pretty easy to add so can implement this if needed.

**Comment by ChihChengLiang** [comment]
@Brechtpd thanks for the update. From #813 it looks like swapping to different implementations is easy. No need to add a configurable keccak bit for now.
